packages/webpack-extraction-plugin/src/babelPluginStripGriffelRuntime.ts

```diff
@@ -173,30 +205,39 @@ export const babelPluginStripGriffelRuntime = declare<
               });
 
               // Returns the styles as a JavaScript object
-              const evaluationResult = argumentPaths[1].evaluate();
+              const evaluationResult = argumentPath.evaluate();
 
               if (!evaluationResult.confident) {
-                throw argumentPaths[1].buildCodeFrameError(
+                throw argumentPath.buildCodeFrameError(
                   [
-                    'Failed to evaluate CSS rules from "__styles" call.',
+                    `Failed to evaluate CSS rules from "${functionKind}" call.`,
                     'Please report a bug (https://github.com/microsoft/griffel/issues) if this error happens',
                   ].join(' '),
                 );
               }
 
-              const cssRulesByBucket = evaluationResult.value as CSSRulesByBucket;
+              if (functionKind === '__styles') {
+                const cssRulesByBucket = evaluationResult.value as CSSRulesByBucket;
+
+                Object.values(cssRulesByBucket).forEach(cssBucketEntries => {
+                  const cssRules = cssBucketEntries.map(cssBucketEntry => {
+                    const [cssRule] = normalizeCSSBucketEntry(cssBucketEntry);
 
-              Object.values(cssRulesByBucket).forEach(cssBucketEntries => {
-                const cssRules = cssBucketEntries.map(cssBucketEntry => {
-                  const [cssRule] = normalizeCSSBucketEntry(cssBucketEntry);
+                    return cssRule;
+                  });
 
-                  return cssRule;
+                  state.cssRules!.push(...cssRules);
                 });
 
-                state.cssRules!.push(...cssRules);
-              });
+                argumentPath.remove();
+              }
 
-              argumentPaths[1].remove();
+              if (functionKind === '__resetStyles') {
+                const cssRules = evaluationResult.value as string[];
+
+                state.cssRules!.push(...cssRules);
+                argumentPaths[2].remove();
```

Comment:
```suggestion
                argumentPaths.remove();
```

you set the argumentPath correctly for both cases

Reply:
Done in 1aeddeb